docs/CSBR.md

```diff
@@ -503,19 +503,16 @@ If the CA decides that the revocation will have an unreasonable impact on its cu
 
 The CA MUST revoke a Code Signing Certificate within one (1) business day if the Subscriber requests in writing that the CA revoke the Certificate or notifies the CA that the original certificate request was not authorized and does not retroactively grant authorization.
 
-#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Malware
+#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Suspect Code
 
-For all incidents involving malware, CAs SHALL revoke the Code Signing Certificate in accordance with and within the following maximum timeframes. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these timeframes.
+Except for cases that fall under Section 4.9.1.1, if, while investigating a problem report, the CA determines the Subscriber's Private Key is likely compromised or being used for Suspect Code, the CA SHALL revoke the corresponding Code Signing Certificate in accordance with and within the following maximum time frames. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these time frames.

```

Comment:
Can we omit the term "likely" from this update? This term introduces a layer of uncertainty or gray area about the key compromise, where it would be much clearer that key compromise must be proven.

Reply:
@chrisbn This does somewhat come back to the point of there being no default way of proving key compromise. However I think it could work to set this at "...is compromised or likely being used for Suspect Code...."
Thoughts?